pyais/messages.py

```diff
@@ -2,22 +2,52 @@
 import json
 import math
 import typing
+import datetime
 from typing import Any, Dict, Optional, Sequence, Union
 
 import attr
 from bitarray import bitarray
 
-from pyais.constants import TalkerID, NavigationStatus, ManeuverIndicator, EpfdType, ShipType, NavAid, StationType, \
-    TransmitMode, StationIntervals
-from pyais.exceptions import InvalidNMEAMessageException, UnknownMessageException, UnknownPartNoException, \
-    InvalidDataTypeException
-from pyais.util import decode_into_bit_array, compute_checksum, get_itdma_comm_state, get_sotdma_comm_state, int_to_bin, str_to_bin, \
```

Comment:
One thing that I notice right away is that you formatted the whole file with black8. While this isn't a problem by itself, this makes it quite hard to focus on the things that really changed.